pal/minigame/taobao.ts

```diff
@@ -1,6 +1,14 @@
-import { IMiniGame } from 'pal/minigame';
+import { IMiniGame, SystemInfo } from 'pal/minigame';
 import { Orientation } from '../screen-adapter/enum-type';
 import { cloneObject } from '../utils';
+import { Language } from '../system-info/enum-type';
+
+//taobao IDE    (language: "Chinese")
+//taobao phone  (language: Andrond: "cn", iPad: 'zh_CN')
+const languageMap: Record<string, Language> = {
+    Chinese: Language.CHINESE,
+    cn: Language.CHINESE,
+};
```

Comment:
need `zh_CN: Language.CHINESE` for iPad ?

Reply:
No need, because `sysytem.lanuage` will be `substr(0,2)`

Reply:
I think it's not a good design for the `pal/minigame` module,

that means when we call function `minigame.getSystemInfoSync().language`
we got `Language.CHINESE` on taobao IDE and Android, but we got 'zh_CN' on iPad